docs/setup/custom-config.mdx

```diff
@@ -51,9 +51,9 @@ First, you should prepare a `config.json` file based on the following template;
         "autoupdate": true
     },
     "auth":{
-        "http_auth_enabled": False,
-        "username": "mindsdb",
-        "password": "123"
+        "http_auth_enabled": True,
```

Comment:
This is False by default so we can keep it as False

Reply:
Done.